dtfs-central-api/src/scheduler/jobs/create-utilisation-reports/index.ts

```diff
@@ -0,0 +1,45 @@
+import { getUtilisationReportDetailsByBankIdAndReportPeriod, saveNotReceivedUtilisationReport } from '../../../services/repositories/utilisation-reports-repo';
+import { getAllBanks } from '../../../services/repositories/banks-repo';
+import { SchedulerJob } from '../../../types/scheduler-job';
+import { Bank } from '../../../types/db-models/banks';
+import { getCurrentReportPeriodForBankSchedule } from '../../../utils/report-period';
+import { asString } from '../../../helpers/validation';
+
+const currentBankReportIsMissing = async (bank: Bank): Promise<boolean> => {
+  const currentReportPeriod = getCurrentReportPeriodForBankSchedule(bank.utilisationReportPeriodSchedule);
+  const currentUtilisationReportForBank = await getUtilisationReportDetailsByBankIdAndReportPeriod(bank.id, currentReportPeriod);
+  return !currentUtilisationReportForBank;
+};
+
+const getBanksWithoutReports = async () => {
+  const banks = await getAllBanks();
+
+  const isMissingBankReport = await Promise.all(banks.map(currentBankReportIsMissing));
+  return banks.filter((_, index) => isMissingBankReport[index]);
```

Comment:
We dont' use `_` for unused variable, please provide a meaningful name to the variable.